akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -484,11 +484,10 @@ private[akka] class ShardRegion(
   // subscribe to MemberEvent, re-subscribe when restart
   override def preStart(): Unit = {
     cluster.subscribe(self, classOf[MemberEvent])
-    if (settings.passivateIdleEntityAfter > Duration.Zero)
-      log.info(
-        "{}: Idle entities will be passivated after [{}]",
-        typeName,
-        PrettyDuration.format(settings.passivateIdleEntityAfter))
+    log.info(
```

Comment:
What will this log as if the user sets it back to off?

Reply:
This was a formatting diff vs a change. The value is either a duration or "off" - as it was doing before.